neurom/fst/_neuritefunc.py

```diff
@@ -318,6 +326,30 @@ def section_radial_distances(neurites, neurite_type=NeuriteType.all, origin=None
     return dist
 
 
+def section_term_radial_distances(neurites, neurite_type=NeuriteType.all, origin=None):
+    '''Get the radial distances of the termination sections for a collection of neurites'''
+    dist = []
+    for n in iter_neurites(neurites, filt=is_type(neurite_type)):
+        pos = n.root_node.points[0] if origin is None else origin
+        dist.extend([sectionfunc.section_radial_distance(s, pos)
+                     for s in iter_sections(n,
+                                            iterator_type=Tree.ileaf,
+                                            neurite_filter=is_type(neurite_type))])
+    return dist
+
+
+def section_bif_radial_distances(neurites, neurite_type=NeuriteType.all, origin=None):
```

Comment:
is this the same as `section_term_radial_distances` except a different iterator? prob. easiest just to have a generic function, and pass in the different iterator types

Reply:
Yes, but I don't want to improve the code with the current PR just extend the functionality that I need, so style improvements can be implemented later?

Reply:
since both pieces of code are being added at the same time, I think it's best to do it now - it will be hard in the future to notice that they are doing almost the same thing

Reply:
The problem with merging those two functions is that then we will need to pass an argument to the "get" instead of having available the 'section_bif_radial_distances' (and respectively the 'term') as a feature. I have no idea how to do this since all the other features are done this way (one function per feature without specified arguments) so if you can propose a fix in order to pass a specific argument in the "get" to create the two required features I can update this.

Reply:
I'd probably do something like:
```
def _section_radial_distances(..., iterator_type):
    ...

def section_term_radial_distances(...):
    return _section_radial_distances(..., iterator_type=Tree.ileaf)

def section_bif_radial_distances(...):
    return _section_radial_distances(..., iterator_type=Tree.ibifurcation_point)
```

This way it's apparent where the bulk of the algorithm is, and how the two functions differ

Reply:
Sure, that certainly works :) Thanks!

Reply:
Sure, if it is something simple there is no problem. Thanks for the suggestion.